lib/pymedphys/_dev/propagate.py

```diff
@@ -303,6 +303,17 @@ def propagate_extras():
                 except KeyError:
                     extras[group] = [key]
 
+    for group, deps in extras.items():
+        extras[group] = sorted(deps)
+
+    extras = tomlkit.item(
+        extras, _parent=pyproject_contents["tool"]["poetry"], _sort_keys=True
+    )
+
+    for _, deps in extras.items():
+        if len(deps.as_string()) > 88:
+            deps.multiline(True)
+
```

Comment:
As a note, the only part of this PR is human written is the above lines. Everything else is auto-generated by `pymedphys dev propagate`